.changelog/2731.txt

```diff
@@ -0,0 +1,23 @@
+```release-note:bug
+resource/mongodbatlas_project: Fixes resource when user doesn't have project owner permission
+```
+
+```release-note:bug
+data-source/mongodbatlas_project: Fixes datasource when user doesn't have project owner permission
+```
+
+```release-note:bug
+data-source/mongodbatlas_projects: Fixes datasource when user doesn't have project owner permission
+```
+
+```release-note:note
+resource/mongodbatlas_project: Deprecates `is_slow_operation_thresholding_enabled`
```

Comment:
should we give some reasoning behind this deprecation?
`property will be supported in a separate resource as it requires different set of permissions.`

Reply:
thanks, change in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2731/commits/9dcf7ff8113db6121602ab88715da2fb8b4a58bb